chapters/0900 Statements.md

```diff
@@ -17,31 +78,317 @@ is intended to describe aspects of Swift analogous to what's found in
 	<p class="first admonition-title">Grammar of expression statement</p>
 	<div class="syntax-group">
 		<p class="syntax-def">
-			<span class="syntax-def-name">
-				<span id="statement_expression">statement</span>
-			</span>
+			<a id="statement_expression"></a> 
+			<span class="name">statement</span>
 			<span class="arrow"> → </span> 
-			<a href="#whitespace-item">expression</a>
+			<a href="0800 Expressions">expression</a>
 			<code>;</code><sub>opt</sub>
 		</p>
 	</div>
 </div>
 
+<p class="item"> </p>
+*See*, <a href="0800 Expressions">Expressions</a>.
 
 #### Declaration Statement
 
-### Compiler Control Statements
+<div class="admonition grammar">
+	<p class="first admonition-title">Grammar of declaration statement</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<a id="statement_declaration"></a>
+			<span class="name">statement</span>
+			<span class="arrow"> → </span> 
+			<a href="1000 Declarations">declaration</a>
+			<code>;</code><sub>opt</sub>
+		</p>
+	</div>
+</div>
+
+<p class="item"> </p>
+*See*, <a href="1000 Declarations">Declarations</a>. 
 
 ### Control Flow Statements
 
-#### Loop Statements
+#### Condition List
+
+A condition list is used in branch statements and while statements to determine 
+the path of execution.
+
+<div class="admonition grammar">
+	<p class="admonition-title">Grammar of a condition list</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<a id="condition-list"></a>
+			<span class="name">condition-list</span>
+			<span class="arrow"> → </span> 
+			<a href="#condition">condition</a>
+			|
+			<a href="#condition">condition</a>
+			<code>,</code>
+			<a href="#condition-list">condition-list</a>
+		</p>
+		<p class="syntax-def">
+			<a id="condition"></a>
+			<span class="name">condition</span>
+			<span class="arrow"> → </span> 
+			<a href="expressions">expression</a>
+			|
+			<a href="#optional-binding-condition">optional-binding-condition</a>
+			|
+			<a href="#case-condition">case-condition</a>
+			|
+			<a href="/inactiveLink">availability-condition</a>
+		</p>
+	</div>
+</div>
+
+<p class="item"> </p>
+A condition list consists of a list of one or more conditions. There are four 
+types of conditions:
+* an expression that evaluates to a `Bool`
+* an `Optional` binding condition
+* a `case` pattern-matching condition
+* a compiler `#availability` condition
+
+<p class="item"> </p>
+A condition list evaluates to `true` if all conditions of the condition list 
+evaluate to `true`.  
+
+<p class="item"> </p>
+Conditions of a condition list are evaluated in sequence, from left to right.  
+If  a condition evaluates to `false`, any remaninig conditions are not evaluated.
+
+<p class="item"> </p>
+If an expression is used as a condition, the expression must evaluate to a 
+`Bool`.  Use of any other type of expression is an error.  [Note: The C `bool` 
```

Comment:
Not objecting too hard, but the note seems a little out of place here.  

Heh, we need chapters for C and Objective C interop (just filed #8 and #9).